spec/diagnose_spec.rb

```diff
@@ -96,7 +95,7 @@
         /  Language: #{@runner.language_name}/,
         /  (Gem|Package) version: #{quoted VERSION_PATTERN}/,
         /  Agent version: #{quoted REVISION_PATTERN}/,
-        /  (Extension|Nif) loaded: true/
+        /  (Extension|Nif) loaded: (t|T)rue/
```

Comment:
Let's call it "Agent booted" or something similar for Python. I don't want the diagnose report to mention the extension if it doesn't have one.

Reply:
We could also omit it entirely because I see it's reading from the agent report, which isn't this section.